cli/ballerina-cli/src/main/java/io/ballerina/cli/task/CreateExecutableTask.java

```diff
@@ -53,29 +53,31 @@ public class CreateExecutableTask implements Task {
     private final transient PrintStream out;
     private Path output;
     private Path currentDir;
+    private Target target;
+    private final boolean isHideTaskOutput;
 
-    public CreateExecutableTask(PrintStream out, String output) {
+    public CreateExecutableTask(PrintStream out, String output, Target target, boolean isHideTaskOutput) {
         this.out = out;
+        this.target = target;
+        this.isHideTaskOutput = isHideTaskOutput;
         if (output != null) {
             this.output = Paths.get(output);
         }
     }
 
     @Override
     public void execute(Project project) {
-        this.out.println();
-
-        if (!project.buildOptions().nativeImage()) {
-            this.out.println("Generating executable");
+        if (!isHideTaskOutput) {
```

Comment:
Why do we need this flag?

Reply:
To maintain the consistency between the old output and the current output. I mean, to avoid printing the `Generating executable` message on the console.